ktor-server/ktor-server-test-suites/jvm/src/io/ktor/server/testing/suites/SustainabilityTestSuite.kt

```diff
@@ -605,6 +607,49 @@ abstract class SustainabilityTestSuite<TEngine : ApplicationEngine, TConfigurati
         }
     }
 
+    @Test
+    @NoHttp2
+    fun testInvalidLineTerminator() = runTest {
+        var rootRouteCalled = false
+        var adminRouteCalled = false
+
+        createAndStartServer {
+            post {
+                rootRouteCalled = true
+                call.respondText { "POST /" }
+            }
+
+            get("/admin") {
+                adminRouteCalled = true
+                call.respondText { "GET /admin" }
+            }
+        }
+
+        socket {
+            getOutputStream().writer().use { writer ->
+                writer.append(
+                    "POST / HTTP/1.1\r\n",
+                    "Host: localhost:8080\r\n",
+                    "Transfer-Encoding: chunked\r\n",
+                    "\r\n",
+                    "5\r\n",
+                    "AAAAA\r2\r\n",
+                    "47\r\n",
+                    "0\r\n\r\n",
+                    "GET /admin HTTP/1.1\r\n",
+                    "Host: localhost:8080\r\n",
+                    "Transfer-Encoding: chunked\r\n",
+                    "\r\n",
+                    "0\r\n\r\n",
+                )
+            }
+            delay(1.seconds)
```

Comment:
Any ideas on how to wait until the server settles down before going to assertions?